io/eolearn/io/geometry_io.py

```diff
@@ -31,26 +91,19 @@ def __init__(self, feature, path, reproject=True, clip=False, config=None, **kwa
         :type feature: (FeatureType, str)
         :param path: A path to a dataset containing vector data. It can be either a local path or a path to s3 bucket
         :type path: str
-        :param reproject: Should the geometries be transformed to EOPatch coordinate reference system?
+        :param reproject: Should the geometries be transformed to coordinate reference system of the requested bbox?
         :type reproject: bool, default = True
-        :param clip: Should the geometries be clipped to EOPatch bbox, or should be geometries kept as they are?
+        :param clip: Should the geometries be clipped to the requested bbox, or should be geometries kept as they are?
         :type clip: bool, default = False
-        :param config: A configuration object containing AWS credentials
+        :param config: A configuration object with AWS credentials (if not provided, ~/.aws/credentials will be used)
         :type config: SHConfig
-        :param kwargs: Additional args that will be passed to `fiona.open` call (e.g. layer='GPKG_LAYER')
```

Comment:
This line that documents `kwargs` has been removed. I would put it back.

Reply:
I would not. 

The base class doesn't need kwargs, and I'd rather that each new implementation of the vector import tasks take care of additional arguments. If - at some point - base task will need kwargs, we can add them.

Reply:
Sorry, I think I marked the wrong class - I meant to say that `VectorImportTask` is now missing documentation of `kwargs`, not `_BaseVectorImportTask`.

Reply:
👍 Added.